service/keyv2/key.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/giantswarm/microerror"
 )
 
+const (
+	CAID  = "ca"
+	CrtID = "crt"
+	KeyID = "key"
+)
+
+const (
+	// ComponentLabel is the label used in the secret to identify a cluster
+	// component.
+	ComponentLabel string = "clusterComponent"
+	// ClusterIDLabel is the label used in the secret to identify a cluster.
+	ClusterIDLabel string = "clusterID"
+)
```

Comment:
This is copy/paste from `certificatetpr`. Good enough for me now. We can boyscout this later.

Reply:
We can take it from `certificates` repo.